src/shared/ranking.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This function get data from localStorage by category name.
+ * @param {string} category
+ * Function return [{username: string, points: number}]
```

Comment:
```suggestion
 * @returns {{username: string, points: number}[]}
```

W JSDoc możemy użyć `@returns` żeby wskazać zwracany typ.

Reply:
zmieniłem, ale w sumie nie wiem czy tak może być jak napisałem teraz?